### ISSUE TITLE
knex init command is looking for a stub knexfile on the wrong directory

### ISSUE THREAD

**Issue by paulo-santana** [knex/knex#4338]
# Environment

Knex version: 0.95.0
Database + version: Not applicable
OS: Debian 10.8 "Buster"

# Bug

Whenever I type `knex init` or `npx knex init` I get an error that says the following:

```
Error: ENOENT: no such file or directory, open '/home/paulo/Documentos/testes-empregos/nave-team/back-end-challenge/node_modules/knex/lib/migrate/stub/knexfile-js.stub'
```

After investigating, I realised the file it's looking for is actually at `node_modules/knex/lib/migrations/migrate/stub/knexfile-js.stub` (note the folder **migrations/** before migrate/

If necessary, I can send a PR to fix this

**Comment by kibertoad** [comment]
PR would be appreciated!

**Comment by kibertoad** [comment]
Released in 0.95.1
